.circleci/config.yml

```diff
@@ -298,9 +298,11 @@ jobs:
       - run_karma:
           disable_synthetic: true
           api_version: 58
+      - run_karma:
+          enable_aria_reflection_global_polyfill: true
       - run_karma:
           disable_synthetic: true
-          disable_aria_reflection_polyfill: true
+          enable_aria_reflection_global_polyfill: true
```

Comment:
I feel we may as well test with synthetic shadow on vs off, especially since this flag will be set to true in LEX.